Assets/_BForBoss/_Weapons/Scripts/BulletBehaviour.ApplyDamage.cs

```diff
@@ -0,0 +1,21 @@
+using Perigon.Entities;
+using UnityEngine;
+
+namespace Perigon.Weapons
+{
+    public interface IApplyDamage
+    {
+        public void HitObject(Collider col);
```

Comment:
NIT, you don't need the `public`

Reply:
Should I just get rid of the interface all together? Wasn't sure if that was a best practice for partial classes

Reply:
So I personally can't currently think of any place that'll need to inject the bullets into. 
So I wouldn't create an interface for this.

Reply:
Sure I'll get rid of it then